dts/bindings/mtd/gd,gd32-nv-flash-v1.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2022 BrainCo Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Flash memory binding of gd32 fmc v1.
+
+  GD32 series use this type flash memory:
+    - GD32C10x
+    - GD32C11x
+    - GD32E10x
+    - GD32E11x
+    - GD32E50x
+    - GD32F3x0
+    - GD32F1x0
+    - GD32E23x
+    - GD32VF103
+    - GD32L23x
+
+include: soc-nv-flash.yaml
+
+compatible: gd,gd32-nv-flash-v1
+
+properties:
+  max-erase-time:
```

Comment:
```suggestion
  max-erase-time-ms:
```

(add units)

Reply:
applied, add uint `ms` to `v1`, `v2` and `v3` max erase time property.